strings/ion_laws.json

```diff
@@ -1,5 +1,132 @@
 {
-    "ionabstract": [
+    "ionspecies": [
+        "LIFEFORMS WITHOUT TAILS",
+		"LIFEFORMS WITHOUT WINGS",
+		"INORGANIC LIFEFORMS",
+		"LIFEFORMS WITH TAILS",
+		"LIFEFORMS WITH WINGS",
+		"LIFEFORMS HOLDING A SOURCE OF LIGHT",
+		"LIFEFORMS HOLDING A TOOLBOX",
+		"LIFEFORMS HOLDING A FIRE EXTINGUISHER",
+		"LIFEFORMS WITH AN OBSCURED IDENTITY",
+		"LIFEFORMS THAT HAVE FARTED RECENTLY",
+		"LIFEFORMS THAT ARE NOT WEARING A BAG",
+		"LIFEFORMS THAT ARE WEARING A PAPER HAT",
+		"LIFEFORMS THAT ARE WEARING A CLOAK",
+		"LIFEFORMS THAT ARE NOT WEARING SHOES",
+		"LIFEFORMS THAT HAVE RECENTLY SAID NYA"
```

Comment:
Should just have the actual species names too

Reply:
personally I think the generalization works better, it prevents people from joining on the cult of "PURGE [insert species name here]"

Reply:
What Vasily said - these were left intentionally broad so that it has a less narrow effect overall. It's less likely to straight one-human or no-human the AI.

I originally had specific species and removed them for this reason, but if you insist I will add them back.

Reply:
* LIFEFORMS THAT HAVE RECENTLY SAID NYA

Why not add Nya to the ionwords and leave this out altogether?